stock_landed_costs_delivery/models/delivery_carrier.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class DeliveryCarrier(models.Model):
+    _inherit = "delivery.carrier"
+
+    lc_product_id = fields.Many2one(
```

Comment:
Is it required to use a product with such flag? As we are programmatically inserting this line, can we use the same product that the delivery carrier already has?

Reply:
I think this was necessary to "separate" if you only wanted to create new lines for only some carriers.

Reply:
Well, I don't see why are you going to put the delivery carrier in the purchase if it's not for this, but if so, better to put a flag for excluding/including them instead of another product.